baiji/config.py

```diff
@@ -76,6 +94,13 @@ def key(self):
     def secret(self):
         return self._try(['AWS_SECRET_ACCESS_KEY', 'AWS_SECRET'], 'AWS_SECRET')
 
+    @property
+    def region(self):
+        return self._try(['AWS_DEFAULT_REGION'], 'REGION')
+
+# Alias to avoid breaking changes
+Credentials = Settings
```

Comment:
I'm wondering if it makes sense to pull out the credentials parsing into a proper `Credentials` class then use it from the the `Settings` class, or even make `Settings` a subclass. Otherwise it really doesn't make sense for this `Credentials` class to exist (strictly as an alias) and we'd want to remove it at some point anyway.

I suppose there's an argument for grouping together multiple breaking changes down the line.